pkg/apis/kops/containerdconfig.go

```diff
@@ -45,6 +47,19 @@ type ContainerdConfig struct {
 	Runc *Runc `json:"runc,omitempty"`
 	// SelinuxEnabled enables SELinux support
 	SeLinuxEnabled bool `json:"selinuxEnabled,omitempty"`
+	// Nri configures the Node Resource Interface.
+	Nri *NRIConfig `json:"nri,omitempty"`
+}
+
+type NRIConfig struct {
+	// Enable NRI support in containerd
+	Enabled *bool `json:"enabled,omitempty"`
```

Comment:
It would be better if this defaulted to `true`, at least if an NRIConfig is specified.